setup.py

```diff
@@ -105,6 +105,7 @@
     # --- `metadata` bundle without records UI -----------------------------
     "invenio-indexer>=1.1.0,<1.2.0",
     "invenio-jsonschemas>=1.0.0,<1.1.0",
+    "invenio-pages>=1.0.0a5",
```

Comment:
very very very minor, for another time: can you maybe move this package a bit below under
`--- extra deps of ILS -----------------------------------------------`?
thanks!